src/ontology/mondo-edit.obo

```diff
@@ -325562,21 +325566,13 @@ property_value: IAO:0006012 "2021-01-01" xsd:string
 
 [Term]
 id: MONDO:0016896
-name: partial deletion of the short arm of chromosome 18
-comment: Reason: duplicate. This will be merged with MONDO:0007800 chromosome 18p deletion syndrome
-subset: disease_grouping
-subset: obsoletion_candidate
-subset: ordo_group_of_disorders {source="Orphanet:261974"}
-synonym: "partial deletion of chromosome 18p" EXACT [Orphanet:261974]
-synonym: "partial deletion of the short arm of chromosome type 18" EXACT [MONDORULE:2, Orphanet:261974]
-synonym: "partial monosomy of chromosome 18p" EXACT [Orphanet:261974]
-synonym: "partial monosomy of the short arm of chromosome 18" EXACT [Orphanet:261974]
-xref: ICD10:Q93.5 {source="ORDO:261974/attributed", source="ORDO:261974/ntbt", source="Orphanet:261974"}
-xref: Orphanet:261974 {source="MONDO:equivalentTo"}
+name: obsolete partial deletion of the short arm of chromosome 18
+xref: Orphanet:261974 {source="MONDO:obsoleteEquivalent"}
```

Comment:
this is where I'm unclear - do we want to leave this here or move it to the new class?